lib/cogctl/actions/relays/delete.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Cogctl.Actions.Relays.Delete do
+  use Cogctl.Action, "relays delete"
+
+  def option_spec do
+    []
+  end
+
+  def run(_options, args,  _config, endpoint),
+    do: with_authentication(endpoint, &do_delete(&1, args))
+
+  defp do_delete(_endpoint, []) do
+    display_arguments_error
+  end
+
+  defp do_delete(endpoint, relay_names) when is_list(relay_names) do
```

Comment:
Is `relay_names` ever going to be anything other than a list? Seems like if it is, that means something's wrong at a higher level (i.e. `run/4` got passed the wrong thing). We could drop the guard here, then.